### PR TITLE
Deconflict multiple endpoint k8s

### PR DESCRIPTION
**Pull request by CollectiveUnicorn**
Adds support for specifying a prefix to the VirtualService so that the same IP can share multiple applications by default

### CODE REVIEW

**Review by gphorvath — APPROVED** [review]

**Review by justinthelaw — COMMENTED** [review]
@CollectiveUnicorn the commits need to be signed, but I can pull this down and test and then rebase under my signature if you aren't able to respond by tomorrow

**Comment by justinthelaw** [comment]
@CollectiveUnicorn both the Transcript upload page, and the AI endpoint on LF-04 are not working after the changes made to LF-UI and here in LF API.

File: chart/values.yaml [hunk 1]
```diff
@@ -1,12 +1,10 @@
-domain: bigbang.dev
-
 image: 
   lfaiAPITag: ###ZARF_CONST_LEAPFROGAI_API_VERSION###
   kiwigridTag: 1.23.3
 
 istio:
   enabled: false
-  gateway: tenant
+  gateway: leapfrogai
```
Comment (YrrepNoj) [inline chart/values.yaml:7]: This should probably be set to `###ZARF_VAR_ISTIO_GATEWAY###`

**Comment by gerred** [comment]
@CollectiveUnicorn when you get a chance, will you discuss some of the intent behind this PR? Prefix routing can be brittle for UIs that aren't aware of a prefix (loading in static assets for say, the UI with everything having a /ui or whatever subroute). What services should this ultimately encompass? Would subdomains be a better long-term option? I think this can make sense but more context is helpful. ty!

**Comment by CollectiveUnicorn** [comment]
@gerred This is particularly for when there is no subdomain. So in the airgap when someone only has an IP address, same for our local environments. If the prefix isn't specified then the UI will eat the curl requests. When we can make subdomains then the prefix doesn't have to be specified.

**Comment by gregclark-defenseunicorns** [comment]
We don't need because of UDS 🪄 
- LFAI team

**Pull request closed without merging**